server.js

```diff
@@ -16,8 +14,11 @@ const handleGetEvents = require('./modules/getEvents');
 
 const app = express();
 app.use(cors());
+app.options('*', cors());  //<--------- Had to add to get auth0 working
```

Comment:
Interesting. Please tell me more about this part when we're both on.  @Kellen-Linse @SpencerTower